src/Header.jsx

```diff
@@ -479,16 +479,92 @@ export default function Header({
               toggleOpen={toggleOpen}
               panelsRef={panelsRef}
             />
-            <Character 
-              open={open}
-              setOpen={setOpen} 
-              toggleOpen={toggleOpen} 
-              panelsRef={panelsRef} 
-              wearActions={wearActions} 
-              previewCanvasRef={previewCanvasRef}
-              game={game}
-            />
-            <World
+            <Tab

```

Comment:
Why is this changed?

Reply:
Will fix, will revert this to main branch

Reply:
It's the linting tool